tests/apis/test_meta_ai.py

```diff
@@ -0,0 +1,90 @@
+# from superai.apis.meta_ai import ModelApiMixin, ProjectAiApiMixin
+# import uuid
+# import pytest
+
+
+# @pytest.fixture()
+# def model_api():
+#     yield ModelApiMixin()
+
+
+# @pytest.fixture()
+# def ai_api():
+#     yield ProjectAiApiMixin()
+
+
+# @pytest.fixture()
+# def existing_app_id():
+#     return "" # Requires actual existing app_id, not committed to Repo
```

Comment:
Why are all the tests disabled?

Reply:
I haven't had VCR setup, but now its working.